_data/past.yml

```diff
@@ -171,12 +176,14 @@
   dates: "August 1-3, 2014"
   url: http://2014.eurucamp.org
   twitter: eurucamp
+  video_link: http://media.eurucamp.org/
 
 - name: JRubyConf EU
   location: Potsdam, Germany
   dates: "August 1, 2014"
-  url: http://jrubyconf.eu/
+  url: http://2014.jrubyconf.eu/
   twitter: jrubyconfeu
+  video_link: http://media.eurucamp.org/
```

Comment:
All eurucamp & jrubyconf archives are at the same page, there is no way to link at a part of it.